fenix/app/src/main/res/values/strings.xml

```diff
@@ -393,21 +394,27 @@
-    <string name="reduce_cookie_banner_unsupported_site_request_submitted">Request to support site submitted</string>
+    <string name="reduce_cookie_banner_unsupported_site_request_submitted" moz:RemovedIn="114" tools:ignore="UnusedResources">Request to support site submitted</string>
+    <string name="reduce_cookie_banner_unsupported_site_request_submitted_2">Support request sent</string>
     <!-- Text for indicating cookie banner handling is currently not supported for this site, this is shown as part of the protections panel with the tracking protection toggle -->
     <string name="reduce_cookie_banner_unsupported_site">Site currently not supported</string>
     <!-- Title text for a detail explanation indicating cookie banner handling is on this site, this is shown as part of the cookie banner panel in the toolbar. The first parameter is a shortened URL of the current site-->
-    <string name="reduce_cookie_banner_details_panel_title_on_for_site">Turn on Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_on_for_site" moz:RemovedIn="114" tools:ignore="UnusedResources">Turn on Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_on_for_site_2">Turn on Cookie banner reduction for %1$s?</string>
     <!-- Title text for a detail explanation indicating cookie banner handling is off this site, this is shown as part of the cookie banner panel in the toolbar. The first parameter is a shortened URL of the current site-->
-    <string name="reduce_cookie_banner_details_panel_title_off_for_site">Turn off Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_off_for_site" moz:RemovedIn="114" tools:ignore="UnusedResources">Turn off Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_off_for_site_2">Turn off Cookie banner reduction for %1$s?</string>
     <!-- Title text for a detail explanation indicating cookie banner reducer didn't work for the current site, this is shown as part of the cookie banner panel in the toolbar.-->
-    <string name="reduce_cookie_banner_details_panel_title_unsupported_site_request">This site is currently not supported by Cookie Banner Reduction. Would you like our team to review this website and add support in the future?</string>
+    <string name="reduce_cookie_banner_details_panel_title_unsupported_site_request" moz:RemovedIn="114" tools:ignore="UnusedResources">This site is currently not supported by Cookie Banner Reduction. Would you like our team to review this website and add support in the future?</string>
+    <string name="reduce_cookie_banner_details_panel_title_unsupported_site_request_2">%1$s can’t automatically reject cookie requests on this site. You can send a request to support this site in the future.</string>
```

Comment:
We need to add a new comment description explaining %1$s

Reply:
Couldn't find this on Figma.

Reply:
<img width="401" alt="Screenshot 2023-05-03 at 09 49 20" src="https://user-images.githubusercontent.com/8118371/235852081-3eb32800-8e54-4317-9256-cb3d75b323fa.png">

Reply:
Thanks!